web/controllers/homepage_controller.ex

```diff
@@ -56,9 +57,9 @@ defmodule App.HomepageController do
 
     content_filter = Enum.filter_map(content, true_tuples, second_value)
 
-    article_query = create_tag_query(tag, "articles")
+    article_query = Resources.create_tag_query(tag, "articles")
 
-    link_query = create_tag_query(tag, "resources")
+    link_query = Resources.create_tag_query(tag, "resources")
```

Comment:
:+1: You're right, I did actually do that here: https://github.com/LDMW/app/pull/101/files#diff-fcc61fedb64f0f4da3b9173fad183d9bR13, but didn't think to go back and change the old one as well. I'll change it